eng/testing/.runsettings

```diff
@@ -10,6 +10,8 @@
     <!-- Configures the architecture of test host. -->
     <TargetPlatform>$$TARGETPLATFORM$$</TargetPlatform>
     <DotNetHostPath>$$DOTNETHOSTPATH$$</DotNetHostPath>
+    <!-- Filter out failing (wrong framwork, platform, runtime or activeissue) tests -->
+    <TestCaseFilter>category!=failing</TestCaseFilter>
```

Comment:
Seems ok as a small incremental change, I wonder if we should add more functionality like runtime has? https://github.com/dotnet/runtime/blob/84f75654f9de8e83784861075ad07529c31310e7/eng/testing/tests.targets#L46-L49

Reply:
I looked at all of those and I don't think that we need any of those.